index.html

```diff
@@ -176,6 +176,19 @@ <h3 class="lang-font">Skills</h3>
           </div>
         </div>
     </section>
+    <section class="contact-section">
+      <h2 class="con-title">Contact me</h2>
+      <p class="con-paragraph">If you have an application you are interested in developing, a feature that
+          you
+          need built or a project that needs coding. I’d love to help with it</p>
+      <form action="https://formspree.io/f/mnqlovzq" method="POST" class="con-form">
+          <input type="text" name="_name" placeholder="Please enter your full name" class="form-text" required>
+          <input type="email" name="_reply" placeholder="Please enter your Email" class="form-text" required>
+          <textarea name="_message" placeholder="Write your message here" maxlength="500"
+              class="form-text message" required></textarea>
+          <button class="con-but" type="submit">Get in touch</button>
+      </form>
+  </section>
```

Comment:
- [ ] It would be better if you stick to the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) design. The style you did is this.
- ![](https://res.cloudinary.com/dflkwvvnz/image/upload/v1629815016/scrnli_8_24_2021_7-21-51_PM_cnkyqs.png)
-   As you see there should be no `grey background` before the form section. Please remove this background.Try to make same as the Figma one 🙏.

Reply:
Done